x-pack/plugins/ml/public/application/components/data_grid/data_grid.tsx

```diff
@@ -166,6 +167,9 @@ export const DataGrid: FC<Props> = memo(
                 />
               );
             },
+            featureInfluence: ({ cellContentsElement }: { cellContentsElement: any }) => (
```

Comment:
Please use `HTMLDivElement ` instead of `any`
```suggestion
            featureInfluence: ({ cellContentsElement }: { cellContentsElement: HTMLDivElement }) => (
```

Reply:
Updated in af2a22dab20501c37e959f5109ac5da42e975692